umm-spec-lib/src/cmr/umm_spec/util.clj

```diff
@@ -31,44 +31,44 @@
   85)
 
 (def valid-url-content-types-map
-  {"DistributionURL" {"GET SERVICE" ["DATACAST URL"
-                                     "EARTHDATA SEARCH"
-                                     "ECHO"
-                                     "EDG"
-                                     "EOSDIS DATA POOL"
-                                     "GDS"
-                                     "GIOVANNI"
-                                     "KML"
-                                     "LAADS"
-                                     "LANCE"
-                                     "LAS"
-                                     "MIRADOR"
-                                     "MODAPS"
-                                     "NOAA CLASS"
-                                     "ON-LINE ARCHIVE"
-                                     "REVERB"]
-                      "GET DATA" ["ACCESS MAP VIEWER"
-                                  "ACCESS MOBILE APP"
-                                  "ACCESS WEB SERVICE"
-                                  "DIF"
-                                  "MAP SERVICE"
-                                  "NOMADS"
-                                  "OPENDAP DATA"
-                                  "OPENDAP DATA (DODS)"
-                                  "OPENDAP DIRECTORY (DODS)"
-                                  "OpenSearch"
-                                  "SERF"
-                                  "SOFTWARE PACKAGE"
-                                  "SSW"
-                                  "SUBSETTER"
-                                  "THREDDS CATALOG"
-                                  "THREDDS DATA"
-                                  "THREDDS DIRECTORY"
-                                  "WEB COVERAGE SERVICE (WCS)"
-                                  "WEB FEATURE SERVICE (WFS)"
-                                  "WEB MAP FOR TIME SERIES"
-                                  "WEB MAP SERVICE (WMS)"
-                                  "WORKFLOW (SERVICE CHAIN)"]}
+  {"DistributionURL" {"GET DATA" ["DATACAST URL"
```

Comment:
I'm very new to the CMR and its taxonomy ... so forgive the naïveté: it looked right to me before? 

Aren't Earthdata Search, ECHO, etc. services? And aren't OpenSearch, WCS, etc. data sources?